File: MODUL-1/python/PRAK0108-231081721007-RaymondHariyono.py
print("diketahui: ")
putaran = float(input("Pak Dengklek mengelilingi taman = "))
jarak = float(input("Jarak tempuh Pak Dengklek = "))

keliling = jarak / putaran
jariJari = keliling / (2 * 3.14)

print("diketahui")
print(f"Pak Dengklek mengelilingi taman {putaran:.0f} putaran")
print(f"jarak tempuh Pak Dengklek {jarak:.0f} kilometer")
print("jawaban :")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jariJari:.2f} kilometer")

File: MODUL-1/bahasa c/PRAK0108-231081721007-RaymondHariyono.c
#include <stdio.h>
int main(){

 float putaran, jarak;
  
  printf("diketahui: \n");
  printf("Pak Dengklek mengelilingi taman: ");
   scanf("%f",& putaran);
   
   printf("Jarak tempuh Pak Dengklek: ");
   scanf("%f", & jarak);

   float keliling = jarak / putaran;
   float jariJari = keliling / (2 * 3.14);
  
   printf("jawaban : \n");
   printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f kilometer ", jariJari);

return 0;
}